tensorboard/plugins/base_plugin.py

```diff
@@ -172,7 +172,8 @@ def __init__(
           handle this function returns must be closed. It is assumed that you
           will pass this file handle to zipfile.ZipFile. This zip file should
           also have been created by the tensorboard_zip_file build rule.
-      data_provider: Instance of `tensorboard.data.provider.DataProvider`.
+      data_provider: Instance of `tensorboard.data.provider.DataProvider`. May
+        be `None` if `flags.generic_data` is set to `"off"`.
```

Comment:
s/"off"/"false"/ I believe?

Reply:
Good catch; thanks. (Originally had `on`/`auto`/`off` but changed it to
`true`/`auto`/`false` per your comment.)

Reply:
FWIW the comment was just illustrative, on/auto/off is fine if you prefer that (and I guess it avoids sounding like stringified booleans).

Reply:
No preference; I’ll keep it as is.